tests/test_multiple_sessions.py

```
@@ -32,15 +32,17 @@ def create_mock_mouse() -> Mock:
     return mock_mouse
 
 
-def mock_metric_fcn_trials(trials, rewarded=None, window=None):
+def mock_metric_fcn_trials(
+    trials: List[TrialSummary], rewarded: bool | None = None
```

Ruff (0.8.2)

Undefined name `List`

(F821)